spec.html

```diff
@@ -15377,11 +15393,52 @@ <h1>Runtime Semantics: Evaluation</h1>
             </table>
           </figure>
         1. Let _r_ be ApplyStringOrNumericBinaryOperator(_lval_, _opText_, _rval_).
-        1. [id="step-assignmentexpression-evaluation-compound-putvalue"] Perform ? PutValue(_lref_, _r_).
+        1. Perform ? PutValue(_lref_, _r_).
         1. Return _r_.
       </emu-alg>
+      <emu-grammar>AssignmentExpression : LeftHandSideExpression `&amp;&amp;=` AssignmentExpression</emu-grammar>
+      <emu-alg>
+        1. Let _lref_ be the result of evaluating |LeftHandSideExpression|.
+        1. Let _lval_ be ? GetValue(_lref_).
+        1. Let _lbool_ be ! ToBoolean(_lval_).
+        1. If _lbool_ is *false*, return _lval_.
+        1. If IsAnonymousFunctionDefinition(|AssignmentExpression|) and IsIdentifierRef of |LeftHandSideExpression| are both *true*, then
```

Comment:
This reads a bit like a garden path sentence to me. I have a slight preference for

```suggestion
        1. If IsAnonymousFunctionDefinition(|AssignmentExpression|) is *true* and IsIdentifierRef of |LeftHandSideExpression| is *true*, then
```

if for no reason other than more consistency in phrasing.

Reply:
Done.